### ISSUE TITLE
SmallCapsFont doesn't select the small cap shape

### ISSUE THREAD

**Issue by simifilm** [wspr/fontspec#73]
<pre>
\documentclass{article}
\usepackage{fontspec}
\usepackage[EU2]{fontenc}
\setsansfont[SmallCapsFont={texgyreadventor-regular}]{texgyreadventor-regular} 
\begin{document}
\sffamily Lalala \textsc{Lala} \\
\end{document}   
</pre>

With this setup (which doesn't make sense but shows the problem with a free font), fontspec does chose the regular font shape instead of the small cap shapes for `\textsc{}`.

With this setup it works:

<pre>
\setsansfont{texgyreadventor-regular} 
</pre>

In other words: fontspec doesn't seem to activate the smcp property of the font defined with SmallCapsFont. It only seems to work for fonts which only have small cap shapes.
